index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>BusMall</title>
+  <link rel="stylesheet" href="css/reset.css">
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <header>
+    <img src="http://via.placeholder.com/150x50" alt="">
+    <h1>BusMall Products</h1>
+    <nav>
+      <ul>
+        <li>Home</li>
+        <li>Products</li>
+        <li>About</li>
+        <li>Contact</li>
+      </ul>
+    </nav>
+  </header>
+  <div class="content">
+    <aside>
+      <h2>Results</h2>
+      <table id="results"></table>
+    </aside>
+    <div class="vote">
+      <section id = "items" class="items">
+          <!-- <img class="item" src="img/bag.jpg" alt="">
+          <img class="item" src="img/banana.jpg" alt="">
+          <img class="item" src="img/bathroom.jpg" alt=""> -->
+      </section>
+      <h2>Click on item to cast your vote!</h2>
+      <section class="text">
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
```

Comment:
I notice you're using lorem ipsum text here; what are you planning to put in this spot? (why is this text here?)